pyclient/README.md

```diff
@@ -62,6 +62,202 @@ $ pip3 install dist/pydeephaven-0.6.0-py3-none-any.whl
 
 ```
 
+## Initialize
+
+The `Session` class is your connection to Deephaven. This is what allows your Python code to interact with a Deephaven server.
+
+```
+from pydeephaven import Session
+
+session = Session()
+```
+
+## Ticking table
+
+The `Session` class has many methods that create tables. This example creates a ticking time table and binds it to Deephaven.
+
+```
+from pydeephaven import Session
+
+session = Session()
+
+table = session.time_table(period=1000000000).update(formulas=["Col1 = i % 2"])
+session.bind_table(name="MyTable", table=table)
+```
+
+This is the general flow of how the Python client interacts with Deephaven. You create a table (new or existing), execute some operations on it, and then bind it to Deephaven. Binding the table gives it a named reference on the Deephaven server.
+
+## Execute a query on a table
+
+`table.update()` can be used to execute an update on a table. This updates a table with a query string.
+
+```
+from pydeephaven import Session
+
+session = Session()
+
+# create a table with no columns and 3 rows
+table = session.empty_table(3)
+# Create derived table having a new column MyColumn populated with the row index "i"
+table = table.update(["MyColumn = i"])
+# Update the Deephaven Web Console with this new table
+session.bind_table(name="MyTable", table=table)
```

Comment:
Conventions question.  Does it make sense for all the bound names in the example to comply with PEP8 (e.g. `my_table`) since this is python?

Reply:
then make the change